src/Phobos.Kafka/src/Petabridge.Phobos.Kafka.Producer/Buffers.cs

```diff
@@ -0,0 +1,520 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Buffers.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2021 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2021 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Akka.Annotations;
+using Akka.Streams;
+
+namespace Petabridge.Phobos.Kafka.Producer
+{
+    /// <summary>
+    /// INTERNAL API
+    /// </summary>
+    /// <typeparam name="T">TBD</typeparam>
+    internal interface IBuffer<T>
```

Comment:
Is this necessary?

Reply:
I wish it isn't, but the Buffer class is an internal class, so I have to copy it over for now.